### PR TITLE
RUM Monitoring Page Performance page

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Creates page about RUM page performance monitoring

### Motivation
DOCS-1360

### Preview

https://docs-staging.datadoghq.com/kari/docs-1360-rum-perf/real_user_monitoring/browser/monitoring_page_performance/

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    text: "Explore your views within Datadog"
+  - link: "/real_user_monitoring/explorer/analytics/"
+    tag: "Documentation"
+    text: "Build analytics upon your events"
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+When a user visits a page on your website, a new **RUM view event** is created. The view contains live page load and performance metrics. Performance metrics are sent to Datadog as the page finishes loading, but might be empty for a few seconds before they appear in the Datadog UI.
```
Comment (hdelaby) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:19]: I would probably say something "Performance metrics are sent to Datadog periodically as the page finishes loading, and might be empty for a few seconds before they appear in the Datadog UI."

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## Identifying performance bottlenecks
+
+To find what's causing a page to under-perform:
+
+1. Set up and configure [RUM Browser monitoring][1] for your application.
+2. Open the [Performance Overview dashboard][2]. It provides you with a high level view of your monitored pages, and you can apply filters to find pages with performance problems.
```
Comment (hdelaby) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:26]: "From the RUM Applications page, head to the Performance Overview dashboard. [...]. Apply filters to narrow your search and uncover performance problems." or something like that.

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+3. When you find a page with problem, click the event to open it in the [RUM Explorer][3].
```
Comment (hdelaby) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:27]: Let's mention:
1. the interaction with the widget
2. the jump from an overview to full granularity of data in the explorer. 

I want to really emphasize on the granularity => that's a key differentiator

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+4. Open the event side panel to see what is causing the performance issue with the help of the waterfall.
```
Comment (hdelaby) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:28]: Clicking on one view in the RUM explorer lets you access performance information and a lot of details about that specific page view. Troubleshoot the performance bottleneck/issue with the waterfall.

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+These performance metrics are key for helping you start your investigations:
+
+- **Time To First Byte:** How long it takes the server to process the request. If the server is slow, try using APM to understand what's causing the slowness on the server side.
+- **First Contentful Paint:** How long until something is displayed. Check the RUM waterfall for blocking resources and long tasks preventing the browser from rendering content.
+- **Loading Time:** How long until the page is interactive. Check the RUM waterfall to see if you're loading too many assets, or if some resources are blocking the rendering.
+
+## All performance metrics
+
+For information about the default attributes for all RUM event types, see [Data Collected][5]. For information about configuring for sampling, global context, or custom user actions and custom errors, see [Advanced Configuration][6]. The following table lists the performance metrics collected from [Navigation Timing API][7] and [Paint Timing API][8]:
```
Comment (hdelaby) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:42]: "The following table lists **Datadog specific metrics along with performance metrics** collected from [Navigation Timing API][7] and [Paint Timing API][8]:" or similar. We also come up with some in-house metrics to help the filtering (error count, action count,...)

**Review by hdelaby — COMMENTED** [review]
Few comments but overall it's really good :) I see there are opportunities to add more link to cross-reference other pages but I leave you with the final word on this. I know there are already a lot of links on this page

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md (hunk 3 above)
Comment (kayayarai) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md:27]: I think I need to see this in action to know what you're talking about enough to write about it. OR, maybe we could record a gif that shows it.

Comment (hdelaby) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md:27]: https://a.cl.ly/BluQppQp

For instance in this GIF, we identify that a group is page is slow in dashboards and then we can look at individual loads of that page in the RUM explorer, look at all the resources loaded etc. Being able to jump from overview to details is key in our strategy :)

Comment (kayayarai) [reply content/en/real_user_monitoring/browser/monitoring_page_performance.md:27]: Perfect. I will put this in words, but i also think I'll add a gif in the docs. This is great, thank you.

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    text: "Explore your views within Datadog"
+  - link: "/real_user_monitoring/explorer/analytics/"
+    tag: "Documentation"
+    text: "Build analytics upon your events"
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+When a user visits a page on your website, a new **RUM view event** is created. The view contains live page load and performance metrics. Performance metrics are sent to Datadog periodically as the page finishes loading, and might be empty for a few seconds before they appear in the Datadog UI.
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:19]:
```suggestion
When a user visits a page on your website, a new **RUM view event** is created. The view contains live page load and performance metrics. Performance metrics are sent to Datadog periodically as the page finishes loading and might be empty for a few seconds before they appear in the Datadog UI.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+## Identifying performance bottlenecks
+
+To find what's causing a page to under-perform:
+
+1. Set up and configure [RUM Browser monitoring][1] for your application.
+2. From the RUM Applications page, open the [Performance Overview dashboard][2]. It provides you with a high level view of your monitored pages. Apply filters to narrow your search and uncover performance problems.
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:26]:
```suggestion
2. From the RUM Applications page, open the [Performance Overview dashboard][2]. It provides you with a high-level view of your monitored pages. Apply filters to narrow your search and uncover performance problems.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+3. When you find a page with problem, dive from the overview dashboard quickly into a RUM view that exemplifies the problem by clicking **View RUM events** to open it in the [RUM Explorer][3], and then clicking a specific user view. The waterfall in the side panel shows performance details, such as load times for resources, that will help you troubleshoot the bottleneck or other issue.
+    {{< img src="real_user_monitoring/browser/rum-page-performance-dive.gif" alt="Diving from Performance Overview dashboard to performance details for a particular view with an issue"  >}}
+
+## Performance metrics for views
+
+Performance metrics are collected for each individual view. These metrics can pinpoint exactly where the problem is occurring when used with the RUM context collected by default (current page view information, geoIP data, browser information, etc.) and extended with [Global Context][4].
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:32]:
```suggestion
Performance metrics are collected for each individual view. These metrics can pinpoint exactly where the problem is occurring when used with the RUM context collected by default (current page view information, GeoIP data, browser information, etc.) and extended with [Global Context][4].
```

I am not positive about this one, but I think this is usually capitalized

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+These performance metrics are key for helping you start your investigations:
+
+- **Time To First Byte:** How long it takes the server to process the request. If the server is slow, try using APM to understand what's causing the slowness on the server side.
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:36]:
```suggestion
- **Time To First Byte:** How long it takes the server to process the request. If the server is slow, try using APM to understand what's causing the slowness on the server-side.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+- **First Contentful Paint:** How long until something is displayed. Check the RUM waterfall for blocking resources and long tasks preventing the browser from rendering content.
+- **Loading Time:** How long until the page is interactive. Check the RUM waterfall to see if you're loading too many assets, or if some resources are blocking the rendering.
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:38]:
```suggestion
- **Loading Time:** How long until the page is interactive. Check the RUM waterfall to see if you're loading too many assets or if some resources are blocking the rendering.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+## Monitoring Single Page Applications
+
+For Single Page Applications (SPAs), the RUM SDK differentiates between `initial_load` and `route_change` navigation with the `loading_type` attribute. If a click on your web page leads to a new page without a full refresh of the page, the RUM SDK starts a new view event with `loading_type:route_change`. RUM tracks page changes using the [History API][14].
+
+Datadog provides a unique performance metric, `loading_time`, which calculates the time needed for a page to load. This metric works for both `initial_load` and `route_change` navigation.
+
+### How is loading time calculated?
+
+To account for modern web applications, loading time watches for network requests and DOM mutations.
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:66]:
```suggestion
To account for modern web applications, loading time watches for network requests, and DOM mutations.
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+- **Initial Load**: Loading Time is equal to _whichever is longer_:
+
+  - The difference between `navigationStart` and `loadEventEnd`.
+  - Or the difference between `navigationStart` and the first time the page has no activity for more than 100ms (activity being defined as ongoing network requests or a DOM mutation).
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:71]:
```suggestion
  - Or the difference between `navigationStart` and the first time the page has no activity for more than 100ms (activity defined as ongoing network requests or a DOM mutation).
```

File: content/en/real_user_monitoring/browser/monitoring_page_performance.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+
+- **SPA Route Change**: Loading Time is equal to the difference between the user click and the first time the page has no activity for more than 100ms (activity being defined as ongoing network requests, or a DOM mutation).
```
Comment (apigirl) [inline content/en/real_user_monitoring/browser/monitoring_page_performance.md:73]:
```suggestion
- **SPA Route Change**: Loading Time is equal to the difference between the user click and the first time the page has no activity for more than 100ms (activity defined as ongoing network requests or a DOM mutation).
```

**Review by apigirl — APPROVED** [review]
just some very small nits

**Pull request merged**